informer/integration/test/collector/init.go

```diff
@@ -17,9 +15,22 @@ import (
 )
 
 var (
-	namespace = "test-informer-collector"
+	namespace = "test-informer-integration-collector"
```

Comment:
This is also aligned with the other tests. I do not understand why we should make it different all the time. We can safe such added complexity.